index-es5.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <title>Chat Adapter Prototype</title>
+  <script src="http://localhost:8080/dist/chat-adapter.es5.js"></script>
+  <script src="https://cdn.botframework.com/botframework-webchat/4.8.0/webchat-es5.js"></script>
```

Comment:
Looks like this needs updating

Reply:
I think Web Chat doesn't upload new builds to cdn and if we want to use their alpha versions we either need to upload by ourselves or use npm

Reply:
Is it a good idea for public repo?

Reply:
What are your concerns?

Reply:
Just if there are any Microsoft policies that might restrict the use of unpkg